### ISSUE TITLE
locus: deleting a locus should check whether there is a collected material that has su ceramic records

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#1565]
deleting a locus checks if any related records exists: small find, collected material, pottery.

It now needs to check if a ceramics preprocessing record exists, too.

**Comment by urapadmin** [comment]
### recording 12.8
- it does that now and if it is the case it calls it "ceramics" while pottery (warecode / su-style fragements) is called "ceramics analysis" here.

**Comment by lbestock** [comment]
@mcornacchia25 this one's for you. The ceramics preprocessing is the "ceramics" tab that is present in the collected material record for bulk: ceramic-diag in the current PVD configuration so you can test it. You know how to log in as admin for deleting.

**Comment by mcornacchia25** [comment]
I can't seem to figure out how to make a new ceramics record, and I don't want to try deleting a locus someone else was testing with and screw them up. What is the process for making a ceramics record that will show up in that ceramics window under registrar? Or is that not what I'm supposed to be doing?

**Comment by urapadmin** [comment]
Poor you, that was not a fair ticket - you had no chance. This part is really chaotic, currently, with all those projects in development at once. I have looked into it and the PVD configuration really did not allow to test this. I changed it and synchronized. This is how it is now: 

If you choose "ceramic - diag." as material for a collected material (or use an existing one of that kind) you find a new tab in collected material: ceramics. It is there that you create ceramic preprocessing entries. And if you delete a locus (or a collected material, in fact) the system has to check whether there is a record in that tab. If so you get a warning otherwise you shouldn't.
